spec/models/device_story_comment_spec.rb

```diff
@@ -9,16 +9,30 @@
   describe '#save' do
     let(:comment) { described_class.new(image: upload_file, content: '_content_', user: user, device_story: device_story) }
 
+    def file_upload(src, content_type, binary: false)
```

Comment:
Why do we need our own upload function? Seems like https://github.com/rails/rails/blob/v6.0.3.7/actionpack/lib/action_dispatch/testing/test_process.rb#L17 supports mime type definition.

Reply:
Ah, nm. I see